stable/chronograf/values.yaml

```diff
@@ -12,7 +12,7 @@ service:
 ## Persist data to a persitent volume
 persistence:
   enabled: false
-  storageClass: generic
+  storageClass: ""
```

Comment:
before this gets split into multiple PRs, could we change these to be commented out and documented to be consistent with other charts?https://github.com/kubernetes/charts/blob/master/stable/redis/values.yaml#L23-L27